kernel/core/os.adb

```diff
@@ -995,15 +995,14 @@ is
 
    procedure os_mbx_send
      (status  : out os_status_t;
-      dest_id :     types.int8_t;
+      dest_id :     os_task_dest_id_t;
```

Comment:
as explained above, we cannot make assumption on the range of the dest_id parameter. I think it should stak plain int8_t.